internal/command/machine/list.go

```diff
@@ -115,16 +118,23 @@ func runMachineList(ctx context.Context) (err error) {
 				}
 			}
 
+			note := ""
+			unreachable := machine.WorkerStatus == "unreachable"
+			if unreachable {
+				unreachableMachines = true
+				note = "*"
+			}
+
 			rows = append(rows, []string{
-				machine.ID,
+				machine.ID + note,
 				machine.Name,
 				machine.State,
 				machine.Region,
-				machine.ImageRefWithVersion(),
-				machine.PrivateIP,
+				lo.Ternary(unreachable, "", machine.ImageRefWithVersion()),
+				lo.Ternary(unreachable, "", machine.PrivateIP),
```

Comment:
no idea why we are nullifying it but this information can come from corrosion if the worker isn't responding